### ISSUE TITLE
radion buttons first click event fired twice

### ISSUE THREAD

**Issue by stebeg** [uikit/uikit#1640]
Hi, 

I found a strange behavior in uikit 2.23.0. When I bind a click event to buttons in a radio button group, the first click event is fired twice. The following click events are fired only once. I guess it's a bug. This can be reprocuced with the following code:

``` html
<div class="uk-margin-top">
  <div id="test-radio-buttons" class="uk-button-group" data-uk-button-radio>
    <button class="uk-button">Button 1</button>
    <button class="uk-button">Button 2</button>
    <button class="uk-button">Button 3</button>
  </div>
</div>
```

``` javascript
$(document).ready(function () {      
  $('#test-radio-buttons > button').click(function () {
    foo();
  });
});

function foo() {
  console.log("radio button clicked");
}
```

**Comment by BernhardBaumrock** [comment]
i had the same issue today, any news on this? i made a quickfix like this:
```
var firstrun = true;
// handle button clicks
tf.find('button').click(function() {
    if(!firstrun) {
        console.log($(this));
    }
    firstrun = false;
});
```

**Comment by philoest** [comment]
We have released the next major version, UIkit3. To start focusing our work on the most recent release, we decided to close all the issues related to UIkit2. Please check, if these issues still persist in version 3 and if so, please open a new issue. 

Thanks for your understanding,
Your UIkit Team

**Comment by jsieler** [comment]
Awesome... Version 3 is not usable, but you already closed all tickets related to version 2. So, what about users who are using version 2 in production. Or your "best" suggestion to put unstable-buggy-feature-less version 3 in production?

**Comment by stebeg** [comment]
> Awesome... Version 3 is not usable, but you already closed all tickets related to version 2. So, what about users who are using version 2 in production. Or your "best" suggestion to put unstable-buggy-feature-less version 3 in production?

Exactly what I think about this. I'm happy that my Project in working with uikit 2.26.4 right now. uikit 3 seems to be very far away from usable. And that's frustrating. That's why I will move to Foundation over the next few weeks.

**Comment by stell** [comment]
This is still relevant. UIkit2 is used for production. UIkit3 is atm only for YT themes and not ready for the outside world yet.
